src/main/java/org/opensearch/index/codec/customcodecs/backward_codecs/lucene912/Lucene912QatCodec.java

```diff
@@ -31,7 +31,10 @@ public enum Mode {
         QAT_LZ4("QATLZ4912", Set.of("qat_lz4")),
 
         /** QAT deflate mode. */
-        QAT_DEFLATE("QATDEFLATE912", Set.of("qat_deflate"));
+        QAT_DEFLATE("QATDEFLATE912", Set.of("qat_deflate")),
```

Comment:
There are no plans to backport any new features to 2.x branches (based on Apache Lucene 9.x). The only codec that needs to have the `QAT_ZSTD` implementation should be `QatZstd101Codec`